### DISCUSSION TITLE
Aspen error FOQUS work around?

### DISCUSSION THREAD

**Discussion by LNemetz** [CCSI-Toolset/FOQUS#1043] [Q&A] [answered]
I currently am having trouble with Aspen licensing. When I open any aspen product some times it get a license and works other times it fails to connect to server and won't open Aspen. My universities IT has reached out to aspen and is waiting on a response but the current work around is to just try opening aspen again and it works. 
When I run UQ in FOQUS and it is able to connect to aspen it works but when fails to connect to aspen the run is given a error and doesn't run. 
Is there a way for FOQUS to use a single Aspen (ACM) session for all the runs in UQ rather than opening Aspen running one run and closing aspen and repeating for next run?

**Comment by boverhof** [comment, marked as answer]
If you specify { "reset":true ... } in the job descriptions the AspenEngine should be left open between runs.
